src/core/warnings.ml

```diff
@@ -1,21 +1,25 @@
 open Ppxlib
 
 type level = Warning | Error
+type kind = ..
 
-type kind =
+type kind +=
   | Unsupported of string
   | Ghost_value of string
   | Ghost_type of string
   | Unsupported_model of string * string
   | Function_without_definition of string
   | Predicate_without_definition of string
 
+exception Unknown_warning
```

Comment:
That’s maybe a bit of a quibble, as we are already defining `Warnings.t` as a `Warning | Error`, but I would prefer `Unknown_kind`.

Reply:
Yes, it would be more consistent, which is always good! :+1: